app/controllers/ProjectSidewalkAPIController.scala

```diff
@@ -139,7 +139,7 @@ class ProjectSidewalkAPIController @Inject()(implicit val env: Environment[User,
         "Neighborhood Name,Label ID,Panorama ID,Attribute Latitude,Attribute Longitude,Label Latitude," +
         "Label Longitude,Heading,Pitch,Zoom,Canvas X,Canvas Y,Canvas Width,Canvas Height,GSV URL,Image Date," +
         "Label Date,Label Severity,Label Temporary,Agree Count,Disagree Count,Not Sure Count,Label Tags," +
-        "Label Description"
+        "Label Description,User Id"
```

Comment:
We've been using 'ID' instead of 'Id' for the rest of the column headers, so let's keep it consistent here.